src/components/NavigationIcon/NavigationIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
+import './NavigationIcon.scss';
+
+interface NavigationIconLinkProps {
+  Icon: OverridableComponent<any>;
+  title: string;
+}
+
+export const NavigationIcon: React.FC<NavigationIconLinkProps> = ({ Icon, title }) => (
```

Comment:
NavigationIcon and NavigationIconLinkProps - it should be NavigationIconProps

I'm sure it was caused by copy paste - when you need to use copy paste it's time for reusing data